cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -1372,12 +1372,13 @@ def play(self,component_name, parameter_name,blocking=True):
 		return ah
 
 	def set_wav_path(self,parameter_name,blocking=True):
+		ah = action_handle("set_wav_path", "set_wav_path", parameter_name, blocking, self.parse)
```

Comment:
Where does this come from?

Reply:
the function returns a `ah` but never initializes it...I copied the initialization from the other functions and adjusted the function strings..